tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_interactive_view.ts

```diff
@@ -68,6 +68,7 @@ export interface TooltipDatum<
   metadata: Metadata;
   closestPointIndex: number;
   point: PointDatum;
```

Comment:
Can we get some consistency in naming to make the code clearer?

`point` and `cursorLocationInDataCoord` live in the same space but that's not clear.
`pixelLocation` and `cursorLocation` live in the same space but that's not clear.

Maybe instead:
`pointInDataCoord` and `cursorInDataCoord` and
`pointInPixelCoord` and `cusorInPixelCoord`?

Or maybe:
`dataPoint` and `dataCursorLocation` and
`pixelPoint` and `pixelCursorLocation`.

Another point of reference for naming is the quartet of functions getDomX, getDomY, getDataX, getDataY.

So that suggests:
`dataPoint` and `dataCursorLocation` and
`domPoint` and `domCursorLocation`.